mods/d2k/rules/player.yaml

```diff
@@ -1,6 +1,8 @@
 Player:
 	AlwaysVisible:
 	TechTree:
+	Production:
+		Produces: Building, Upgrade
```

Comment:
Why did you add `Building` here?
It seems to work fine (even better) without it.